build.sbt

```diff
@@ -171,6 +171,7 @@ lazy val codegenJavaCompilationTest = project
   .settings(
     (publish / skip) := true,
     name := "akkaserverless-codegen-java-compilation-tests",
+    Compile / PB.protoSources += baseDirectory.value / ".." / ".." / "proto-test-files",
```

Comment:
Are we going to do something similar like this in the scripted test to include them? Have you tried that it works?